05_syntax_lab/05.py

```diff
@@ -1,5 +1,14 @@
-"""
-Write a program that randomizes integers in a loop
-until it finds a number that is divisable by: 7, 13 and 15
-"""
-
+from random import randint
+index = 1
+x = randint(1,1000000)
+while index != 0:
+	index = index + 1
+	if x % 7 == 0:
+		if x % 13 == 0:
+			if x % 15 == 0:
+				print "Bingo!"
+				print "This is the number: %s" % x
+				index = 0
+	x = randint(1,1000000)
+	
+

```

Comment:
* Why do you need the ```index``` here?
* The 3 conditions create a strange looking steps. In such cases usually it is better to have a single ```if``` conditions. Can you do that?
* Could you eliminate the duplicate call to ```randint```?